### PR TITLE
Improve c sharp

### PR DESCRIPTION
**Pull request by theHamsta**
Address #1349

### CODE REVIEW

**Comment by theHamsta** [comment]
The `@` syntax is not supported by the parser. Please open an issue there!

File: queries/c_sharp/highlights.scm [hunk 1]
```diff
@@ -6,14 +6,18 @@
 (method_declaration
   type: (identifier) @type)
 
-; This would be nice but fails in some cases
-; https://github.com/nvim-treesitter/nvim-treesitter/pull/203
```
Comment (theHamsta) [inline queries/c_sharp/highlights.scm:10]: this PR has been merged.

**Comment by theHamsta** [comment]
image

File: queries/c_sharp/highlights.scm (hunk 1 above)
Comment (theHamsta) [reply queries/c_sharp/highlights.scm:10]: Let's hope the parser has improved since then.

**Review by vigoux — APPROVED** [review]

**Comment by theHamsta** [comment]
Todo: add namespaces

**Pull request merged**